builtin/providers/azurerm/encryption_settings.go

```diff
@@ -0,0 +1,163 @@
+package azurerm
```

Comment:
Currently there are no acceptance tests, because getting an end to end to work via automation is a lot of work and would also take a long time to run due to encryption, or else would require having a pre-encrypted OS image that is hosted somewhere permanent (and still be slow to copy).

What do you guys do in this situation?